### PR TITLE
Dark Mode Update

### PR DESCRIPTION
**Pull request by Ambikeshishere**
Adding Dark Mode 
Add a new Course Python Programming

### CODE REVIEW

**Comment by PulseZest** [comment]
if possible then add this type of icon Button in the nav bar section 🌙 and then implement the dark.
@Divyansh200600  @Ambikeshishere  discuss and implement this features

**Comment by im-divxnshh** [comment]
Sir i know dark mode is cool but on this page I think dark mode is ruined the beuty of page so I dont add dark mode

**Comment by PulseZest** [comment]
@Divyansh200600  When the dark mode implement nicely then I am very sure,  it will never be ruined.
You can also try to fix this by creating a new branch and name darkmode_fix let's see how make this great.

File: index.html [hunk 1]
```diff
@@ -129,6 +146,17 @@ <h3 class="feature-title">Fast-paced Courses</h3>
 
   <div class="container overflow-hidden text-center">
     <div class="row gx-6">
+      <div class="col">
```
Comment (PulseZest) [inline index.html:152]: The Image of the Python is not showing ? May be the path of the image is not correct or may be the image is not added in the images folder

**Review by PulseZest — APPROVED** [review]
Approved this time, But you have to work on this in future.

File: index.html (hunk 1 above)
Comment (im-divxnshh) [reply index.html:152]: @PulseZest ok sir now you tell me it works, So I definetly work on darkmode

**Pull request merged**